spras/analysis/summary.py

```diff
@@ -50,8 +51,44 @@ def summarize_networks(file_paths: Iterable[Path], node_table: pd.DataFrame, alg
         number_edges = nw.number_of_edges()
         ncc = nx.number_connected_components(nw)
 
+        # Save the max/median degree, average clustering coeffienct, and density
+        if number_nodes <= 1 or number_edges == 0:
+            max_degree = 0
```

Comment:
`max_degree` and other degree statistics will be incorrectly reported as zero if we consider self loops. Is this a problem?

Reply:
The idea that a self loop gets represented as an indegree of 2 for an undirected edge needs to be thought through more. Should we also also report # of self loops?